### ISSUE TITLE
Chip Component

### ISSUE THREAD

**Issue by andrew-frueh** [HealthCatalyst/Fabric.Cashmere#118]
Primarily used in conjunction with a filter drawer, but there may be other uses as well.  I think the [Angular material version](https://material.angular.io/components/chips/overview) gives us a pretty good model to follow.  As in that component, having a standard chip and an "input chip" directive that can be closed is probably a good idea.

![chipexample](https://user-images.githubusercontent.com/22795893/36118834-858929dc-0ffb-11e8-970a-593a3275702e.png)

**Comment by jtrujill** [comment]
I'll take this on. Should be easy

**Comment by andrew-frueh** [comment]
Hey Jeremy, I actually got this one started last night.  It’s about halfway done, but I wanted to wait until the big layout change PR got merged before I went too far with it.  Hopefully you haven’t put too much work into it yet.


ANDREW FRUEH | DIRECTOR OF USER EXPERIENCE | HealthCatalyst
3165 MILLROCK DR #400  |  SALT LAKE CITY  |  UT  |  84121
o. 855.309.6800 | c. 801.696.9520
e. andrew.frueh@healthcatalyst.com<mailto:dale.sanders@healthcatalyst.com> | w. www.healthcatalyst.com<http://www.healthcatalyst.com/>





On Feb 19, 2018, at 3:09 PM, Jeremy Trujillo <notifications@github.com<mailto:notifications@github.com>> wrote:


I'll take this on. Should be easy
